app/code/Magento/Sales/Test/Unit/ViewModel/CreditMemo/Create/ItemsToRenderTest.php

```diff
@@ -135,7 +139,7 @@ public function testGetItems(): void
         $this->converter->method('itemToCreditmemoItem')
             ->willReturn($this->creditmemoItemParent);
 
-        $this->assertEquals(
+        $this->assertLessThanOrEqual(
```

Comment:
I would revert these changes to keep the original coverage and add one more test case for the newly added code.

Reply:
@ihor-sviziev Thanks for your update